experimental/rocm/rocm_driver.c

```diff
@@ -51,6 +52,11 @@ static iree_status_t iree_hal_rocm_driver_create_internal(
     iree_string_view_t identifier,
     const iree_hal_rocm_driver_options_t* options,
     iree_allocator_t host_allocator, iree_hal_driver_t** out_driver) {
+  // Hack to force device kernel arguments when available and improve kernel
+  // latency. There doesn't seem to be any API to enable this. This option will
+  // become the default in ROCm 6.1.
+  // TODO: Remove this after upgrading to ROCm 6.1.
+  setenv("HIP_FORCE_DEV_KERNARG", "1", /*replace=*/0);
```

Comment:
you'll want to guard this on `#if defined(IREE_PLATFORM_LINUX)` (no setenv on windows)

Reply:
ohh, I thought it should be available everywhere given it comes form stdlib.h. Fixed now.

Reply:
Ah, getenv is and setenv is not.